fiftyone/utils/openworld_sam.py

```
@@ -0,0 +1,324 @@
+    unit passed through the DataLoader and into :meth:`OpenWorldSAMModel._predict_all`.
+
+    Args:
+        device: the :class:`torch:torch.device` to move tensors to
+        using_half_precision (False): whether to cast tensors to fp16
+    """
+
+    def __init__(self, device, using_half_precision=False):
+        self._device = device
+        self._using_half_precision = using_half_precision
+
+    def __call__(self, img):
```

Ruff (0.15.20)

[warning] Missing return type annotation for special method `__call__`

(ANN204)